user-guide/ssh-access.rst

```diff
@@ -67,6 +67,12 @@ copying files from and to the EC2 instance appears unnecessary hard at first.
 
 Luckily OpenSSH's ``scp`` supports jump hosts with the ``ProxyCommand`` configuration option:
 
+.. code-block:: bash
+
+    $ scp -o ProxyCommand="ssh -W %h:%p odd@odd-eu-west-1.myteam.example.org" mylocalfile.txt ubuntu@172.31.146.1:
```

Comment:
Note you can also use the `-J` flag like this:

```sh
scp -J odd@odd-eu-west-1.myteam.example.org mylocalfile.txt ubuntu@172.31.146.1:
```

Reply:
Which version do you have? `-J` isn't available for my scp. However, I could reduce it a little bit based on https://www.madboa.com/blog/2017/11/02/ssh-proxyjump/

Reply:
I see, I have this version:

```
ssh -V
OpenSSH_8.3p1, OpenSSL 1.1.1h  22 Sep 2020
```

But better to use the version that works with older versions ofc.

Reply:
@mikkeloscar Thanks for checking. For reference mine is:

```
OpenSSH_7.6p1 Ubuntu-4ubuntu0.3, OpenSSL 1.0.2n  7 Dec 2017
```

Let's keep it at the more compatible syntax for now.